tests/test_utils.py

```diff
@@ -528,6 +529,18 @@ def test_undefined_pickle():
     assert undefined2 is Undefined
 
 
+def test_on_lower_camel_zero_length():
+    assert to_lower_camel('') == ''
+
+
+def test_on_lower_camel_one_length():
+    assert to_lower_camel('a') == 'a'
+
+
+def test_on_lower_camel_many_length():
+    assert to_lower_camel('i_like_turtles') == 'iLikeTurtles'
+
+
```

Comment:
could be written as follows

```suggestion
@pytest.mark.parametrize(
    'value, expected_result',
    [('', ''), ('a', 'a'), ('i_like_turtles', 'iLikeTurtles')],
)
def test_on_lower_camel(value: str, expected_result: str):
  assert to_lower_camel(value) == expected_result
```